docs/reference/quickstart/full-text-tutorial.asciidoc

```diff
@@ -0,0 +1,428 @@
+[[full-text-tutorial]]
+== Basic searching and filtering in {es}
+++++
+<titleabbrev>Basics: Searching and filtering</titleabbrev>
+++++
+
+This quick start guide is a hands-on introduction to filtering, searching, and aggregrating data using the `_search` API and QueryDSL.
+
+[discrete]
+[[full-text-tutorial-create-index]]
+=== Create an index
+
+We'll use a cooking blog dataset in this tutorial.
+Create the `cooking_blog` index and its mapping to get started.
+
+[source,console]
+----
+PUT /cooking_blog
+{
+  "mappings": {
+    "properties": {
+      "title": {
+        "type": "text",
+        "analyzer": "standard" <1>
+      },
+      "description": {
+        "type": "text"
+      },
+      "author": {
+        "type": "text",
+        "fields": {
+          "keyword": { <2>
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "date": {
+        "type": "date",
+        "format": "yyyy-MM-dd"
+      },
+      "category": {
+        "type": "keyword" 
+      },
+      "tags": {
+        "type": "keyword"
+      },
+      "rating": {
+        "type": "float"
+      }
+    }
+  }
+}
+----
+// TESTSETUP
+<1> The `standard` analyzer is used by default if an `analyzer` isn't specified.
+Note that analyzers are determined per field. Each field can have a different analyzer.
+<2> Uses <<multi-fields,multi-fields>> to index `author` as both `text` and `keyword` types. This will enable both full-text search and strict term matches.
+
+[TIP]
+====
+Full-text search is powered by <<analysis,text analysis>>. 
+Text analysis normalizes and standardizes text data so it can be efficiently stored in an inverted index and searched in near real-time.
+Analysis happens at both <<analysis-index-search-time,index and search time>>.
+====
+
+[discrete]
+[[full-text-tutorial-index-data]]
+=== Add sample blog posts to your index
+
+Next, index some example blog posts.
+
+[source,console]
+----
+POST /cooking_blog/_bulk
```

Comment:
Would this make more sense in a notebook?

Reply:
You mean the whole tutorial? For sure, the plan is to have notebook versions too once have a few of these :)